cyclonedds_vendor/CMakeLists.txt

```diff
@@ -3,12 +3,21 @@ project(cyclonedds_vendor)
 find_package(ament_cmake REQUIRED)
 
 include(ExternalProject)
-ExternalProject_Add(cyclonedds
-  GIT_REPOSITORY "https://github.com/eclipse-cyclonedds/cyclonedds.git"
-  CMAKE_ARGS
-   -DBUILD_CONFTOOL:BOOL=OFF
-   -DBUILD_IDLC:BOOL=OFF
-   -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}
-  )
+set(cyclonedds_vendor CACHE BOOL "Should we download and build cycloneDDS from source?")
+
+if (cyclonedds_vendor)
```

Comment:
There should not be space between `if` and `(` or may cause lint error like [this](http://build.ros2.org/job/Eci__nightly-cyclonedds_ubuntu_bionic_amd64/lastCompletedBuild/testReport/rmw_cyclonedds_cpp/lint_cmake/whitespace_extra__CMakeLists_txt_36_/).